iotfunctions/__init__.py

```diff
@@ -1,3 +1,13 @@
+# *****************************************************************************
+# © Copyright IBM Corp. 2018.  All Rights Reserved.
```

Comment:
Is it supposed to say 2018 even if it's a new copyright entry ?

Reply:
The file is old. The head was forgotten to be added when the other files got their headers.